docs/developers-guide/driver-changelog.md

```diff
@@ -24,6 +24,8 @@ title: Driver interface changelog
 
 - Removed `driver/set-database-used`. Drivers should set default databases in their connection specs instead.
 
+- Added multi-method `metabase.driver/compile-rename-table` for drivers to generate SQL to rename tables.
```

Comment:
I assume you'll rebase this PR on master and use `metabase.driver/rename-tables!` instead